src/app/rpmostree-override-builtins.cxx

```diff
@@ -90,6 +96,59 @@ handle_override (RPMOSTreeSysroot  *sysroot_proxy,
 
   g_autoptr(GVariant) previous_deployment = rpmostree_os_dup_default_deployment (os_proxy);
 
+  if (override_replace && opt_ex_pin_from_repos)
+    {
+      override_replace_final = g_ptr_array_new_with_free_func (free);
+      g_autoptr(GPtrArray) queries = g_ptr_array_new ();
+
+      for (const char *const* it = override_replace; it && *it; it++)
+        {
+          auto pkg = *it;
+
+          if (rpmostreecxx::is_http_arg (pkg) || rpmostreecxx::is_rpm_arg (pkg))
+            {
+              g_ptr_array_add (override_replace_final, (gpointer)g_strdup (pkg));
+              continue;
+	    }
+          else
+            {
+              g_ptr_array_add (queries, (gpointer)pkg);
+            }
+        }
+      if (queries->len > 0)
+        {
+          g_ptr_array_add (queries, NULL);
+          g_autofree char *refresh_transaction_address = NULL;
+
+          if (!rpmostree_os_call_refresh_md_sync (os_proxy, options, &refresh_transaction_address,
+                                                 cancellable, error))
+            return FALSE;
+          if (!rpmostree_transaction_get_response_sync (sysroot_proxy, (const char*)refresh_transaction_address,
+                                                       cancellable, error))
+            return FALSE;
+          if (!rpmostree_osexperimental_call_download_packages_sync (osexperimental_proxy,
+                                                                     (const char *const*)queries->pdata,
+                                                                     NULL, &list_fds,
+                                                                     cancellable, error))
+            return FALSE;
+
+          if (list_fds == NULL)
+            {
+              printf ("No matching package names at enabled repositories\n");
```

Comment:
`g_print`

Reply:
Honestly I'm fine with `printf` too; I don't think we use any of the glib formatting infrastructure, so it just boils down to a wrapper for `printf`.

Reply:
(But the consistency argument definitely applies)